server/config/config.js

```diff
@@ -0,0 +1,21 @@
+require('dotenv').config();
+
+module.exports = {
+  development: {
+    username: process.env.DEV_USER,
+    password: process.env.DEV_PASS,
+    database: process.env.DEV_DB,
+    host: process.env.DEV_HOST,
+    dialect: 'postgres',
+  },
+  test: {
```

Comment:
I don't think this should be here?

Reply:
Ok, where can I put it, as the real values are taken from an env file available locally and can be made available in travis, heroku environment

Reply:
You may need to rethink environment variable names. 

Since you're using a similar database in all environments, it makes sense to me to have a single environment variable for the database connection. Each environment will set to a different value. e.g.

`DATABASE_URL`